src/store/store.ts

```diff
@@ -31,13 +31,17 @@ interface Actions {
   resetStore: () => void
   updateExploreProjectId: (id: number) => void
   updateCurrentProject: (project: CurrentProjectResponse) => void
+  updateIsQuiet: (newStatus: boolean) => void
+  updateIsPaused: (newStatus: boolean) => void
 }
 
 const initialState: State = {
   user: {
     id: undefined,
     username: undefined,
     email: undefined,
+    birth_date: "01:01:1973",
```

Comment:
Should we have default values? I mean I like the ones you chose. But maybe undefined is better?

Reply:
yep, set these to undefined now